README.md

```diff
@@ -2,6 +2,20 @@
 
 Collections is a prototype project at the Center for Open Science. This project is experimental, scope, technologies, code and functionality may change. 
 
+# Login Configuration
+1. Create a developer app at [https://staging.osf.io/settings/applications/](https://staging.osf.io/settings/applications/) with the following settings:
+    * Project homepage URL: http://localhost:8000/
+    * Callback URL: http://localhost:8000/accounts/osf/login/callback/
+
+2. In the django admin panel, create a new Site with domain name: http://localhost:8000/
+    * Note: The site id must match SITE_ID defined in the django app settings
```

Comment:
This could be extended a bit more. For instance

 ```Go to  localhost:8000/admin to view admin panel and login. In the admin panel, click add Site and create a new site with domain name: ...." 

```side id must match SITE_ID variable defined in {ADD PATH here}```